### PR TITLE
implement 1st year/05 Binary Search &06 Prime number python solution

### PR DESCRIPTION
**Pull request by VickyGuo0907**
### Have you read the [Contributing Guidelines on Pull Requests]()?

- [X] Yes

### Description

(Write your answer here.)

### Checklist

- [X] I've read the contribution guidelines.
- [X] This PR meets all the requirements like Clean Code, proper README and Documentation.

### Related Issues or Pull Requests

Fixes: #5

### Screenshots (if any)

 Original           | Updated
 :--------------------: |:--------------------:
 **original screenshot** | **updated screenshot**|

### CODE REVIEW

File: 1st Year/06. Prime Number/prime_no.py [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+  return True
+
+
+if __name__ == "__main__":
+
+  num = int(input("Enter the number: "))
+  result = is_prime(num)
+
+  if result:
+    print(num, " is a prime.")
```
Comment (atarax665) [inline 1st Year/06. Prime Number/prime_no.py:32]:
```suggestion
    print(num, "is a prime.")
```

File: 1st Year/05. Binary Search in 1D Array/Binary_search_in_1D_array.py [hunk 2]
```diff
@@ -0,0 +1,57 @@
+## To implement binary search technique in 1D Array
+from bisect import bisect_left
```
Comment (atarax665) [inline 1st Year/05. Binary Search in 1D Array/Binary_search_in_1D_array.py:2]: Do not use in-built function ```bisect_left()```. You are expected to implement the traditional binary search algorithm.

File: 1st Year/05. Binary Search in 1D Array/Binary_search_in_1D_array.py [hunk 3]
```diff
@@ -0,0 +1,57 @@
...
+  for i in range(0, num):
+    item = int(input())
+    lst_input.append(item)
+
+  key = int(input("Enter the key to be searched: "))
+
+  result = binary_search(lst_input, key)
+
+  if result != -1:
+    print("Key is available at {} location.".format(result + 1))
```
Comment (atarax665) [inline 1st Year/05. Binary Search in 1D Array/Binary_search_in_1D_array.py:27]:
```suggestion
    print("Key is available at index {}.".format(result + 1))
```

**Review by atarax665 — CHANGES_REQUESTED** [review]

File: 1st Year/06. Prime Number/prime_no.py (hunk 1 above)
Comment (VickyGuo0907) [reply 1st Year/06. Prime Number/prime_no.py]: solved

File: 1st Year/05. Binary Search in 1D Array/Binary_search_in_1D_array.py (hunk 3 above)
Comment (VickyGuo0907) [reply 1st Year/05. Binary Search in 1D Array/Binary_search_in_1D_array.py]: solved

File: 1st Year/05. Binary Search in 1D Array/Binary_search_in_1D_array.py (hunk 2 above)
Comment (VickyGuo0907) [reply 1st Year/05. Binary Search in 1D Array/Binary_search_in_1D_array.py]: solved

**Review by atarax665 — APPROVED** [review]
:+1: Great work, code is well structured and clean.

**Pull request merged**